bundle/jsky.app.ot/src/main/resources/jsky/app/ot/gemini/altair/AltairForm.jfd

```diff
@@ -169,7 +169,7 @@
       </void> 
       <void method="add"> 
        <object class="com.jformdesigner.model.FormComponent"> 
-        <string>jsky.util.gui.OptionWidget</string> 
+        <string>javax.swing.JRadioButton</string>
```

Comment:
I've been wondering for awhile why we still have these and if they are ever used, since JFormDesigner is no longer used, and I believe requires a license that we don't even presently have, if memory serves.